infra/terraform/modules/data_access/organization_events/organization_events.py

```diff
@@ -0,0 +1,53 @@
+import json
+import logging
+import os
+
+import boto3
+
+
+'''
+Environment variables:
+ - CREATE_ROLE_ARN, ARN of the lambda function to create the IAM role
+ - DELETE_ROLE_ARN, ARN of the lambda function to delete the IAM role
+ - LOG_LEVEL, change the logging level (default is "DEBUG"). Must be one of
+   the python logging supported levels: "CRITICAL", "ERROR", "WARNING",
+   "INFO" or "DEBUG" (See: https://docs.python.org/2/library/logging.html#logging-levels)
+'''
+
+
+LOG = logging.getLogger(__package__)
+LOG_LEVEL = os.environ.get("LOG_LEVEL", "DEBUG")
+LOG.setLevel(LOG_LEVEL)
+
+
+def event_received(sns_event, context):
+    EVENT_HANDLERS = {
+        "member_added": os.environ["CREATE_ROLE_ARN"],
+        "member_removed": os.environ["DELETE_ROLE_ARN"],
+    }
+
+    LOG.debug("SNS event received = {}".format(json.dumps(sns_event)))
```

Comment:
The reason why I started to do it is because AWS CloudWatch logs shows JSON nicely with syntax highlighting, etc...which is good for debugging.

It's just a minor thing but it's nicer :)